packages/xrui/core/WebLayer.ts

```diff
@@ -25,6 +34,37 @@ export class WebLayer {
     target: false
   }
 
+  //Only supports digits right now
+  //TO DO: Add alphabetical range support, specific characters
+  prerasterizedRange = [] as number[]
+  prerasterizedImages: Map<number, string> = new Map()
+
+  async prerasterizeRange() {
+    this.manager.prerasterized = true
```

Comment:
this is a global state; won't this have side-effects for things that aren't prerasterized ?

Reply:
Yes, I'm going to change this to a parameter that can be passed into the manager's functions as an argument.